reactioncommerce/reaction
# (fix: 3619) Fix for: Shop switcher opens off-screen

Resolves #3619  
Impact: major
Type: bugfix

## Issue
In a marketplace setup with many shops the dropdown popover can go offscreen completely if the window height is small enough.

## Solution
This PR prevents the shop switch dropdown to go off screen. The tether
library has an option for this : "pin". That makes sense in this case,
because the nav bar is also fixed and won't go off-screen. Not sure
if it makes sense to put this option to all dropdowns in general.

## Breaking changes
none expected

## NOTICE

This PR requires  [#3617](https://github.com/reactioncommerce/reaction/pull/3725/files) to be applied.

## Testing
1. As an admin, set up marketplace and add approx. 10 shops
2. Activate them
3. Navigate to index route
4. Reduce window height
5. Click on shop selector
6. Dropdown list should not go offscreen
7. Dropdown should be scrollable
